independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/strategy/CodestartFileStrategy.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.devtools.codestarts.strategy;
+
+import java.util.Objects;
+import java.util.function.Predicate;
+
+public class CodestartFileStrategy implements Predicate<String> {
```

Comment:
what is this used for ?

Reply:
the strategy defined in the codestart.yml is transformed to a list of `CodestartFileStrategy`, then we use it to test processed files and find the matching handler.